components/sections/ProductsSection.vue

```diff
@@ -9,11 +9,22 @@
     >
       <li v-for="product in products" :key="product.id">
         <ProductCard
-          class="max-w-92 mt-5 lg:max-w-72"
-          :navigate-url="`/products/${product.id}`"
-          :image-url="product.images[0]"
-          :title="product.title"
+          class="max-w-96"
+          :key="product._id"
+          :name="product.name"
+          :description="product.description"
           :price="product.price"
+          :category="product.category"
+          :image="product.images[0]"
```

Comment:
shouldn't this be an array?

Reply:
I think that,

it's image of product card and should be one image from the figma,
But when we use current product page there we should show the list of images in slider

Reply:
ah that's true